next-i18next.config.js

```diff
@@ -0,0 +1,16 @@
+const IS_DEVELOPMENT = process.env.NODE_ENV === 'development';
```

Comment:
We already have this kind of constant in [configs/constants.js](https://github.com/Ukraine-Relief-Efforts/ukraine-info-guide/blob/4443bfab938507161d3b9d1d0e24eb217a54ee5e/configs/constants.js)

Reply:
The problem is that [configs/constants.js](https://github.com/Ukraine-Relief-Efforts/ukraine-info-guide/blob/4443bfab938507161d3b9d1d0e24eb217a54ee5e/configs/constants.js) (like all of our code) is a JS module, but in this config file we can only use `require()`/`module.exports` and not `import`/`export`.

I'm going to go ahead and merge this now, but if you've got a better solution please feel free to make a ticket for it!

Thanks for the review!